exonum-java-binding-common/src/main/proto/block.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+
+option java_package = "com.exonum.binding.common.serialization";
+option java_outer_classname = "BlockProtos";
+
+message Block {
+  int32 proposerId = 1;
+  int64 height = 2;
```

Comment:
We use uint64. [blockchain.proto](https://github.com/exonum/exonum/blob/master/exonum/src/encoding/protobuf/proto/blockchain.proto)

I thought that if protobuf specification allows this type than we should use it. But as I understood from [protocol-buffers/docs](https://developers.google.com/protocol-buffers/docs/proto#scalar) it may cause some inconvenience for Java developers when dealing with large unsigned numbers.

Also I wonder why do you write your own incompatible .proto files?
Maybe we can plan on how to sync them in future. It is also important for service/light client developers since they need .proto files as a part of exonum interface.

Reply:
Yes, I wanted to reuse proto files from core, but I wasn't sure about how to sync. Current block.proto and transactionLocation.proto are temporary, we'll definitely have to use the same proto files

Reply:
Thanks, that's a good question if we can pull these files automatically/have an integration or system test verifying they are up-to-date. @dip56 :question:

> inconvenience … with large unsigned numbers.

The question is how likely are any of the corresponding objects that use uint64 can exceed 2^63-1 (9.223372036854775807 × 10^18). If they can't — it is not a problem.

@MakarovS , if those are temporary, please mark them as such and add an issue to this epic. Or, if they are ready in the core, we may pull them from here.

Reply:
I refactored those so we have similar protos now